### ISSUE TITLE
tea: is directory: /Users/james/go

### ISSUE THREAD

**Issue by magnusviri** [teaxyz/cli#335]
First time I run `tea go` it works. The second time it fails. I don't know much about installing go packages except it installs everything into ~/go. So I don't know what's going on here.

```
> tea go install github.com/maaslalani/confetty@latest
go: downloading github.com/maaslalani/confetty v0.0.0-20221105190856-6c6f1b5b605f
go: downloading github.com/charmbracelet/harmonica v0.2.0
go: downloading github.com/charmbracelet/lipgloss v0.6.0
go: downloading github.com/charmbracelet/bubbletea v0.22.2-0.20221016150627-cbe309d6241c
go: downloading github.com/mattn/go-runewidth v0.0.14
go: downloading github.com/muesli/termenv v0.13.0
go: downloading github.com/muesli/reflow v0.3.0
go: downloading github.com/containerd/console v1.0.3
go: downloading github.com/mattn/go-isatty v0.0.16
go: downloading github.com/mattn/go-localereader v0.0.1
go: downloading github.com/muesli/ansi v0.0.0-20211031195517-c9f0611b6c70
go: downloading github.com/muesli/cancelreader v0.2.2
go: downloading golang.org/x/term v0.0.0-20221017184919-83659145692c
go: downloading github.com/rivo/uniseg v0.4.2
go: downloading github.com/aymanbagabas/go-osc52 v1.2.1
go: downloading github.com/lucasb-eyer/go-colorful v1.2.0
go: downloading golang.org/x/sys v0.1.0
> tea go install github.com/fogleman/primitive@latest
tea: is directory: /Users/james/go
```

If I delete the ~/go directory, then it works.

```
> tea go install github.com/fogleman/primitive@latest
go: downloading github.com/fogleman/primitive v0.0.0-20200504002142-0373c216458b
go: finding module for package github.com/nfnt/resize
go: finding module for package github.com/fogleman/gg
go: finding module for package github.com/golang/freetype/raster
go: finding module for package golang.org/x/image/math/fixed
go: downloading github.com/fogleman/gg v1.3.0
go: downloading github.com/golang/freetype v0.0.0-20170609003504-e2365dfdc4a0
go: downloading golang.org/x/image v0.3.0
go: downloading github.com/nfnt/resize v0.0.0-20180221191011-83c6a9932646
go: found github.com/nfnt/resize in github.com/nfnt/resize v0.0.0-20180221191011-83c6a9932646
go: found github.com/fogleman/gg in github.com/fogleman/gg v1.3.0
go: found github.com/golang/freetype/raster in github.com/golang/freetype v0.0.0-20170609003504-e2365dfdc4a0
go: found golang.org/x/image/math/fixed in golang.org/x/image v0.3.0
```

**Comment by magnusviri** [comment]
In fact, just `tea go` produces the error.

```
> tea go
tea: is directory: /Users/james/go
```

If I specify the full path to go, it works fine.

```
/opt/tea/go.dev/v1.19.4/bin/go
Go is a tool for managing Go source code.

Usage:

	go <command> [arguments]
...
```

**Comment by mxcl** [comment]
```sh
cd /
tea go
```

You have a directory called `go` in your home directory.

Obviously this is a bug, but that is what is happening.

**Comment by mxcl** [comment]
Fixed in 0.22
